### DISCUSSION TITLE
Instalation help

### DISCUSSION THREAD

**Discussion by yeeterwa** [Serene-Arc/bulk-downloader-for-reddit#468] [Q&A]
Can someone please just help me with installing. Im a complete noob and know nothing of this, I'm completely clueless. Can someone help me install it and stuff. I already have python installed. I keep getting syntax error but I know nothing so I really have no idea how to fix it

**Comment by aliparlakci** [comment]
Have you read the README? If you have, tell us where you are puzzled and need help. If you haven't, please read it before your next reply.

**Reply by yeeterwa** [reply]
I have read it but the commands dont seem to work. When I open python and put in the command it just says syntax error and nothing more

**Comment by aliparlakci** [comment]
You don't open python. You open a Command Prompt or Powershell and type in
the commands there.

**Reply by yeeterwa** [reply]
I managed to download some photos from a test subreddit but I don't know where they are placed. I tried the command directory and when I type something next to it, it says unexpected argument

**Reply by Serene-Arc** [reply]
It says unexpected argument because you are not supplying all of the arguments. Please read the README *carefully* and try the example commands given. If it still fails, post the command that you are trying so we can see what you're doing wrong. A generic error doesn't really help.

**Reply by yeeterwa** [reply]
That did the trick, I just didnt specify enough where it should download. Thank you a lot it works perfectly
